packages/ahrocksdb/ahrocksdb.0.2.0/opam

```diff
@@ -39,9 +39,16 @@ depends: [
 build: [
   ["dune" "subst"] {dev}
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest"] {with-test}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 x-ci-accept-failures: ["debian-unstable"]
```

Comment:
I'm not sure it's worth patching packages especially if the latest version fixed the issue already. I'd recommend just ignoring the failure instead
```suggestion
x-ci-accept-failures: ["debian-11" "debian-unstable"]
```